src/store/models/ChartPrices/ChartPrices.ts

```diff
@@ -1,5 +1,5 @@
-import { MONTHS } from "@config/months";
-import { WEEKDAYS } from "@config/weekdays";
+import { MONTHS } from "config/months";
+import { WEEKDAYS } from "config/weekdays";
 
 export type ChartPricesApi = number[];
```

Comment:
возможно, было бы лучше расписать тип подробнее, наподобие:
```suggestion
export type ChartPricesApi = [timestamp: number, price: number];
```